pkg/tfbridge/log.go

```diff
@@ -114,3 +118,49 @@ func (lr *LogRedirector) Write(p []byte) (n int, err error) {
 
 	return written, nil
 }
+
+// hostLogger is parameterized by L to allow `Status` to return the native Log type of a
+// stable Logger.
+type hostLogger struct {
+	ctx  context.Context
+	host *provider.HostClient
+	urn  resource.URN
+
+	status bool
+
+	err *multierror.Error
```

Comment:
The err accumulator is eyebrow-raising. Why would you do that? What kind of errors are you trying to accumulate here?

Reply:
Why ever collect them if we don't want to expose them? What do you want to do with them? Can they just be flushed to glog and forgotten?